core-tests/src/test/java/org/mule/runtime/core/util/ClassUtilsTestCase.java

```diff
@@ -186,37 +190,37 @@ public void testLoadingResourceEnumeration() throws Exception
     @Test
     public void testGetSatisfiableMethods() throws Exception
     {
-        List methods = ClassUtils.getSatisfiableMethods(FruitBowl.class, new Class[]{Apple.class}, true,
+        List methods = ClassUtils.getSatisfiableMethods(FruitBowl.class, new Class[] {Apple.class}, true,
```

Comment:
Don't change code style on code that is not related to the main commit, but if you want to do it... add static imports